workitem/link/link_repository_blackbox_test.go

```diff
@@ -0,0 +1,133 @@
+package link_test
+
+import (
+	"context"
```

Comment:
Built-in context is the standard library. I don't see any major drawback in using it.